app/src/main/java/uw/hcrlab/kubi/lesson/prompts/NamePrompt.java

```diff
@@ -70,6 +110,17 @@ public void afterTextChanged(Editable s) {
     }
 
     public void handleResults(Result res) {
+        NameResult result = (NameResult) res;
 
+        if(result.hasBlame()) {
+            Toast toast = Toast.makeText(getActivity().getApplicationContext(), result.getBlame(), Toast.LENGTH_SHORT);
+            toast.show();
+        }
```

Comment:
do you think Toast will be the final way that we show blame text? Or is this a placeholder for the future way we'll show blame text?